addons/pos_event/data/point_of_sale_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="pos_category_event" model="pos.category">
+            <field name="name">Events</field>
+            <field name="image_128" type="base64" file="point_of_sale/static/img/misc_category.png" />
```

Comment:
Maybe there is a better image that can be used here

Reply:
YEEES SIR